volume/drivers/aws/aws.go

```diff
@@ -238,28 +244,50 @@ func (d *Driver) Create(
 		DryRun:           &dryRun,
 		Encrypted:        &encrypted,
 		Size:             &sz,
-		Iops:             iops,
 		VolumeType:       volType,
 		SnapshotId:       snapID,
 	}
+	// Gp2 Volumes don't support the iops parameter
+	if *volType != opsworks.VolumeTypeGp2 {
+		req.Iops = iops
+	}
 	vol, err := d.ec2.CreateVolume(req)
 	if err != nil {
 		dlog.Warnf("Failed in CreateVolumeRequest :%v", err)
 		return "", err
 	}
+
+	volumeID := *vol.VolumeId
```

Comment:
why do this? remove, and remove extra newlines, you are still using both `volume.Id` and `volumeID` below anyways